src/dotnetsheff.Api/KeepAlive/KeepAlive.cs

```diff
@@ -1,14 +0,0 @@
-using Microsoft.Azure.WebJobs;
```

Comment:
Why you removed this file?

Reply:
Forgot it when moving files on new AzFunc v2 proj. Added it back